tests/Fixtures/app/AppKernel.php

```diff
@@ -96,10 +97,11 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
 
         $loader->load(__DIR__."/config/config_{$this->getEnvironment()}.yml");
 
+        $alg = class_exists(SodiumPasswordEncoder::class) && SodiumPasswordEncoder::isSupported() ? 'auto' : 'bcrypt';
```

Comment:
Uhh... Symfony really screwed this one up. :x

Reply:
It's not bcrypt if sodium isn't supported but the native encoder. I do not know if it's important here.
The bcrypt encoder is now deprecated

Reply:
It's only for old versions so I think we don't care (this code will go be thrown away at some point).